live_meetings.md

```diff
@@ -0,0 +1,23 @@
+With thanks to the Portland Python User Group - https://www.meetup.co...
```

Comment:
did you want this to trail off into an elipsis?

Reply:
no, copypasta error, I'll address